plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/client/BackpressureRestClient.java

```diff
@@ -51,9 +59,13 @@ public BackpressureRestClient(RestClient delegate, ElasticsearchConfig config)
                     Throwable cause = executionAttemptedEvent.getLastFailure();
                     if (cause != null && cause instanceof ResponseException) {
                         Response response = ((ResponseException) cause).getResponse();
-                        log.warn("Failed HTTP response: %s", response);
+                        log.debug("Failed HTTP response: %s", response);
                     }
-                });
+                    stopwatch.get().start();
```

Comment:
How do you know that retry will start immediately after `onFailedAttempt`?
I don't see we measure retry time in other places.

Could number of retires instead

Reply:
The Failsafe docs say this:
```
Registers the listener to be called when an execution attempt fails. You can also use onFailure to determine when the execution attempt fails and and all retries have failed.
```
I think this means it will be called as soon as failure is discovered. This means the stopwatch will be in STARTED state while the backpressure delay is applied. It will accumulate the delay times from all the attempts this thread went through. The delay doesn't have to start immediately after the listener is called, but the time for any calculations done by Failsafe will be insignificant compared to the delay times (nanoseconds vs. seconds).